apps/chat/src/utils/app/conversation.ts

```diff
@@ -237,53 +237,3 @@ export const groupModelsAndSaveOrder = (
 
   return result;
 };
-
-export const addPausedError = (
```

Comment:
Why did you remove this code?

Reply:
probably a skip during the pull, I returned the code back